### PR TITLE
bpo-35849: Added thousands separators to Lib/pstats.py final report

### PR DESCRIPTION
**Pull request by evandrocoan**
https://bugs.python.org/issue35849

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](https://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for your contribution, we look forward to reviewing it!

File: Lib/pstats.py [hunk 1]
```diff
@@ -416,7 +416,7 @@ def print_call_heading(self, name_size, column_title):
                 subheader = isinstance(value, tuple)
                 break
         if subheader:
-            print(" "*name_size + "    ncalls  tottime  cumtime", file=self.stream)
+            print(" "*name_size + "       ncalls     tottime     cumtime", file=self.stream)
```
Comment (eamanu) [inline Lib/pstats.py:419]: Why don't replace it for `\t\t`?

Comment (evandrocoan) [reply Lib/pstats.py:419]: Because I had completely forgot about using tabs. But, indeed, here tabs should excel their best.

**Comment by brettcannon** [comment]
Thanks for the PR, but closing as the CLA has not been signed within the last month. If you do decide to sign the CLA we can re-open this PR.

File: Lib/pstats.py (hunk 1 above)
Comment (evandrocoan) [reply Lib/pstats.py:419]: Actually, I just tried that out, and they aligned very off.

**Pull request closed without merging**